js/memory-matching.js

```diff
@@ -1,22 +1,60 @@
-
-
 var matchingGame = {
-	elapsedTime: 0
+    elapsedTime: 0
+  , github: new GitHub()
 };
 
 matchingGame.deck = []
 
-// Like this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice
+function getHubbers(callback) {
+   matchingGame.github.get("orgs/github/members", { all: true }, function (err, data) {
+        if (err) {
+            console.warn(err);
+            callback(null, { hubbers: window.Hubbers });
+        } else {
+            var complete = 0;
+            for (var i = 0; i < data.length; ++i) {
+                (function (cHubber) {
+                    matchingGame.github.get("users/" + cHubber.login, function (err, user) {
+                        if (err) {
+                            console.warn(err);
+                            if (++complete === data.length) {
+                                callback(null, { hubbers: window.Hubbers });
+                            }
+                        } else {
+                            cHubber.name = user.name;
+                            if (++complete === data.length) {
+                                callback(null, { hubbers: data });
+                            }
+                        }
+                    });
+                })(data[i]);
+            }
+        }
+    });
+}
+
+// http://stackoverflow.com/a/2450976/1420197
+function shuffle(array) {
+    var currentIndex = array.length, temporaryValue, randomIndex ;
 
-Hubbers["hubbers"].sort(shuffle).slice(0,8).map(function(hubber){
-	matchingGame.deck.push(hubber)
-	matchingGame.deck.push(hubber)
```

Comment:
@alysonla Wowow, always learning something: this can be written as `matchingGame.deck.push(hubber, hubber)`. :laughing: